### PR TITLE
 ✅ more tests for extract output

### PR DESCRIPTION
**Pull request by fiendish**
also adds the keep_map shortcut operation

### CODE REVIEW

**Comment by fiendish** [comment]
I also threw in a working demo configuration for boyd17. files are in https://s3.console.aws.amazon.com/s3/buckets/kf-study-us-east-1-prd-sd-p445achv/study-files/modified/

File: tests/data/test_boyd_study/extract_configs/Boyd_Minimal_Data_Elements_Cuellar_9-2-2018_EDITED_v2_map.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
+from common import constants
+from etl.extract.operations import *
+from etl.transform.standard_model.concept_schema import CONCEPT
+
+source_data_url = (
+    'file:///Users/kelmana/Documents/kids-first/kf-study-imports/Avi/'
```
Comment (znatty22) [inline tests/data/test_boyd_study/extract_configs/Boyd_Minimal_Data_Elements_Cuellar_9-2-2018_EDITED_v2_map.py]: This should change to the S3 url for the file. I tried doing that but things didn't work and then I realized that we need #80 to merge first.

Comment (fiendish) [reply tests/data/test_boyd_study/extract_configs/Boyd_Minimal_Data_Elements_Cuellar_9-2-2018_EDITED_v2_map.py]: lol yeah. I'm filing a bug. Trust me that it runs for now and I'll update it after these all merge.

Comment (znatty22) [reply tests/data/test_boyd_study/extract_configs/Boyd_Minimal_Data_Elements_Cuellar_9-2-2018_EDITED_v2_map.py]: k 👍

**Pull request merged**